Script/AtomicEditor/editor/Editor.ts

```diff
@@ -123,15 +143,12 @@ class Editor extends Atomic.ScriptObject {
 
     // event handling
     handleExitRequested(data) {
-        Preferences.getInstance().write();
-        EditorUI.shutdown();
-
-    }
-
-    handleEditorEventQuit(data) {
-
-        this.sendEvent("ExitRequested");
-
+      this.subscribeToEvent("AllResourceEditorsClosed", (_) => {
```

Comment:
AllResourceEditorsClosed event sent only when project close or editor close requested, so it's not gonna be fired during normal use. But, hm, I can't imagine another name for that event